Topics/Tech_Stacks/JSONRPC.md

```diff
@@ -0,0 +1,25 @@
+### JSON-RPC
```

Comment:
# JSON-RPC

It should be the biggest header if it's at the beginning of the page, so maybe use just one hashtag